src/Modules/OrchardCore.Commerce.Payment.Stripe/Extensions/PaymentExtensions.cs

```diff
@@ -44,8 +43,8 @@ paymentMethod.Type switch
             _ => paymentMethod.Type,
         };
 
-    public static IPayment CreatePayment(this PaymentIntent paymentIntent, Amount amount) =>
-        new Payment.Models.Payment(
+    public static Commerce.Abstractions.Models.Payment CreatePayment(this PaymentIntent paymentIntent, Amount amount) =>
```

Comment:
maybe adding `using Commerce.Abstractions.Models;` better?

Reply:
Can't, it will cause this exception:
> Error CS0118 : 'Payment' is a namespace but is used like a type

Reply:
Uh, that because there is a folder/namespace names using singular form instead of plural. Payments vs Payment.

Okay